nototools/noto_cmap_phase3.xml

```diff
@@ -106,7 +106,7 @@
     <tr>Prti,Inscriptional Parthian,34,0000 000d 0020 00a0 10b40-10b55 10b58-10b5f,-1,</tr>
     <tr>Rjng,Rejang,58,0000 000d 0020 002c 002e 003a 00a0 00d7 200b-200d 2012-2015 2022 25cc 25fb-25fe a930-a953 a95f,-1,</tr>
     <tr>Runr,Runic,93,0000 000d 0020 00a0 16a0-16f8,-1,</tr>
-    <tr>SYM2,Symbols2,1880,0000-0020 0023 002a 0030-0039 007f-00a0 20e3 2400-2426 2440-244a 2700-2704 2706-2709 270c-2727 2729-274b 274d 274f-2752 2756-2794 2798-27af 27b1-27be 2800-28ff 2b00-2b57 2b59-2b73 2b76-2b95 2b98-2bb9 2bbd-2bc8 2bca-2bd1 2bec-2bef 4dc0-4dff fff9-fffb 10140-1018e 10190-1019b 101a0 101d0-101fd 102e0-102fb 10e60-10e7e 1d300-1d356 1d360-1d371 1f000-1f02b 1f030-1f093 1f0a0-1f0ae 1f0b1-1f0bf 1f0c1-1f0cf 1f0d1-1f0f5 1f321-1f32c 1f336 1f37d 1f394-1f39f 1f3cb-1f3ce 1f3d4-1f3df 1f3f1-1f3f3 1f3f5-1f3f7 1f43f 1f441 1f4fd-1f4fe 1f53e-1f54a 1f54f 1f568-1f579 1f57b-1f594 1f597-1f5a3 1f5a5-1f5fa 1f650-1f67f 1f6c6-1f6cb 1f6cd-1f6cf 1f6e0-1f6ea 1f6f0-1f6f3 1f780-1f7d4 1f800-1f80b 1f810-1f847 1f850-1f859 1f860-1f887 1f890-1f8ad 1f93b 1f946,-1,</tr>
+    <tr>SYM2,Symbols2,1972,0000-0020 0023 002a 0030-0039 007f-00a0 20e3 23f1-23f3 2400-2426 2440-244a 2700-2704 2706-2709 270c-2727 2729-274b 274d 274f-2753 2756-2794 2798-27af 27b1-27be 2800-28ff 2b00-2b57 2b59-2b73 2b76-2b95 2b98-2bb9 2bbd-2bc8 2bca-2bd1 2bec-2bef 4dc0-4dff fff9-fffb 10140-1018e 10190-1019b 101a0 101d0-101fd 102e0-102fb 10e60-10e7e 1d300-1d356 1d360-1d371 1f000-1f02b 1f030-1f093 1f0a0-1f0ae 1f0b1-1f0bf 1f0c1-1f0cf 1f0d1-1f0f5 1f30d-1f30f 1f315 1f31c 1f321-1f32c 1f336 1f378 1f37d 1f393-1f39f 1f3a7 1f3ac-1f3ae 1f3c2 1f3c4 1f3c6 1f3ca-1f3ce 1f3d4-1f3e0 1f3ed 1f3f1-1f3f3 1f3f5-1f3f7 1f408 1f415 1f41f 1f426 1f43f 1f441-1f442 1f446-1f449 1f44d-1f44e 1f453 1f46a 1f47d 1f4a3 1f4b0 1f4b3 1f4bb 1f4bf 1f4cb 1f4da 1f4df 1f4e4-1f4e6 1f4ea-1f4ed 1f4f7 1f4f9-1f4fb 1f4fd-1f4fe 1f508 1f50d 1f512-1f513 1f53e-1f54a 1f54f-1f579 1f57b-1f594 1f597-1f5a3 1f5a5-1f5fa 1f650-1f67f 1f687 1f68d 1f691 1f694 1f698 1f6ad 1f6b2 1f6b9-1f6ba 1f6bc 1f6c6-1f6cb 1f6cd-1f6cf 1f6e0-1f6ea 1f6f0-1f6f3 1f780-1f7d4 1f800-1f80b 1f810-1f847 1f850-1f859 1f860-1f887 1f890-1f8ad 1f93b 1f946,-1,</tr>
```

Comment:
Not related to this change. I wonder if we can change the format of this file to list one range / one code point per line. That way, 'diff' would be more 'readable'.